docs/connectors.rst

```diff
@@ -80,6 +80,24 @@ facebook developer portal.
 Slack Setup
 -----------
 
+.. note::
+
+   **How to get the Slack credentials:** You need to set up a Slack app.
+
+      1. To create the app go to: https://api.slack.com/apps and click on *"Create New App"*.
+      2. Activate the following features: interactive components, event subscriptions, bot users,
+         permissions (for basic functionality you should subscribe to the ``message.channel``,
+         ``message.groups``, ``message.im`` and ``message.mpim`` events)
+      3. The ``slack_channel`` is the target your bot posts to. This can be a channel,
+         an app or an individual person
+      4. Use the entry for ``Bot User OAuth Access Token`` in the "OAuth & Permissions" tab
+         as your ``slack_token``
+
+
+   For more detailed steps, visit the
+   `slack api docs <https://api.slack.com/incoming-webhooks>`_.
```

Comment:
can we actually just move this note back to where it was? looks a bit strange at the start of the section imo.

Reply:
I didn't modify this note, where was it before?

Reply:
ok got this setup back to the way it was